evaluator/evaluator_test.go

```diff
@@ -660,7 +651,7 @@ func TestBuiltinFunctions(t *testing.T) {
 		{`find([1,2], f(x) {x == "some"})`, nil},
 		{`arg("o")`, "argument 0 to arg(...) is not supported (got: o, allowed: NUMBER)"},
 		{`arg(3)`, ""},
-		{`pwd().split("").reverse().slice(0, 33).reverse().join("").replace("\\", "/", -1)`, "github.com/abs-lang/abs/evaluator"}, // Little trick to get travis to run this test, as the base path is not /go/src/
+		{`pwd().split("").reverse().slice(0, 33).reverse().join("").replace("\\", "/", -1)`, "/evaluator"}, // Little trick to get travis to run this test, as the base path is not /go/src/
```

Comment:
I'm a little bit worried about changing all tests to `Contains`. Maybe it's worth isolating, or commenting this test out.

Reply:
There are at least a couple of cases where I think that the `error == expected` testing paradigm does not work well: 

1) environment dependent tests such as `pwd()` and `$(command)` can be retained as long as we match the  important parts of the results, and
2) tests that contain error location lines that are not essential to the test result.

By using `strings.Contains(error, expected)` we can focus the match on the critical part of the result or even the entire result if we wish. But if we always use `error == expected` we are limiting our flexibility.